### ISSUE TITLE
Use ClassName::class syntax throughout the Bundle

### ISSUE THREAD

**Issue by hanzi** [sonata-project/SonataMediaBundle#1331]
As mentioned in https://github.com/sonata-project/SonataMediaBundle/pull/1325#issuecomment-345026083 we want to switch to `ClassName::class` syntax instead of spelling out fully-qualified class names manually, e.g. `"Sonata\Foo\Bar\ClassName"`.

**Comment by greg0ire** [comment]
Here are the usages I found:

<details>

```

grep -rn '\w\\' src/**/*.php|grep -v 'use '|grep -v 'namespace '
src/Admin/BaseMediaAdmin.php:112:                if (method_exists('Symfony\Component\HttpFoundation\JsonResponse', 'transformJsonError')) {
src/Admin/BaseMediaAdmin.php:191:        $hiddenType = method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')
src/Admin/BaseMediaAdmin.php:192:            ? 'Symfony\Component\Form\Extension\Core\Type\HiddenType'
src/Admin/BaseMediaAdmin.php:209:            $modelListType = method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')
src/Admin/BaseMediaAdmin.php:210:                ? 'Sonata\AdminBundle\Form\Type\ModelListType'
src/Admin/GalleryAdmin.php:120:        $choiceType = method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')
src/Admin/GalleryAdmin.php:121:            ? 'Symfony\Component\Form\Extension\Core\Type\ChoiceType'
src/Admin/GalleryAdmin.php:125:        $collectionType = method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')
src/Admin/GalleryAdmin.php:126:            ? 'Sonata\CoreBundle\Form\Type\CollectionType'
src/Admin/GalleryHasMediaAdmin.php:40:        $modelListType = method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')
src/Admin/GalleryHasMediaAdmin.php:41:            ? 'Sonata\AdminBundle\Form\Type\ModelListType'
src/Admin/GalleryHasMediaAdmin.php:45:        $hiddenType = method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')
src/Admin/GalleryHasMediaAdmin.php:46:            ? 'Symfony\Component\Form\Extension\Core\Type\HiddenType'
src/Admin/ORM/MediaAdmin.php:33:        $choiceType = method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')
src/Admin/ORM/MediaAdmin.php:34:            ? 'Symfony\Component\Form\Extension\Core\Type\ChoiceType'
src/Admin/ORM/MediaAdmin.php:63:        $ormChoiceType = method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')
src/Admin/ORM/MediaAdmin.php:64:            ? 'Sonata\DoctrineORMAdminBundle\Filter\ChoiceFilter'
src/Block/FeatureMediaBlockService.php:49:        if (method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')) {
src/Block/FeatureMediaBlockService.php:50:            $immutableArrayType = 'Sonata\CoreBundle\Form\Type\ImmutableArrayType';
src/Block/FeatureMediaBlockService.php:51:            $textType = 'Symfony\Component\Form\Extension\Core\Type\TextType';
src/Block/FeatureMediaBlockService.php:52:            $textareaType = 'Symfony\Component\Form\Extension\Core\Type\TextareaType';
src/Block/FeatureMediaBlockService.php:53:            $choiceType = 'Symfony\Component\Form\Extension\Core\Type\ChoiceType';
src/Block/GalleryBlockService.php:130:        if (method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')) {
src/Block/GalleryBlockService.php:131:            $modelListType = 'Sonata\AdminBundle\Form\Type\ModelListType';
src/Block/GalleryBlockService.php:132:            $immutableArrayType = 'Sonata\CoreBundle\Form\Type\ImmutableArrayType';
src/Block/GalleryBlockService.php:133:            $textType = 'Symfony\Component\Form\Extension\Core\Type\TextType';
src/Block/GalleryBlockService.php:134:            $choiceType = 'Symfony\Component\Form\Extension\Core\Type\ChoiceType';
src/Block/GalleryBlockService.php:135:            $numberType = 'Symfony\Component\Form\Extension\Core\Type\NumberType';
src/Block/GalleryBlockService.php:136:            $checkboxType = 'Symfony\Component\Form\Extension\Core\Type\CheckboxType';
src/Block/GalleryListBlockService.php:63:        if (method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')) {
src/Block/GalleryListBlockService.php:64:            $immutableArrayType = 'Sonata\CoreBundle\Form\Type\ImmutableArrayType';
src/Block/GalleryListBlockService.php:65:            $textType = 'Symfony\Component\Form\Extension\Core\Type\TextType';
src/Block/GalleryListBlockService.php:66:            $integerType = 'Symfony\Component\Form\Extension\Core\Type\IntegerType';
src/Block/GalleryListBlockService.php:67:            $choiceType = 'Symfony\Component\Form\Extension\Core\Type\ChoiceType';
src/Block/MediaBlockService.php:106:        if (method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')) {
src/Block/MediaBlockService.php:107:            $immutableArrayType = 'Sonata\CoreBundle\Form\Type\ImmutableArrayType';
src/Block/MediaBlockService.php:108:            $textType = 'Symfony\Component\Form\Extension\Core\Type\TextType';
src/Block/MediaBlockService.php:109:            $choiceType = 'Symfony\Component\Form\Extension\Core\Type\ChoiceType';
src/Block/MediaBlockService.php:237:        $modelListType = method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')
src/Block/MediaBlockService.php:238:            ? 'Sonata\AdminBundle\Form\Type\ModelListType'
src/Command/CleanMediaCommand.php:38:            ->addOption('dry-run', null, InputOption::VALUE_NONE, 'Execute the cleanup as a dry run. This doesn\'t remove any files');
src/Controller/Api/GalleryController.php:78:     *  output={"class"="Sonata\DatagridBundle\Pager\PagerInterface", "groups"={"sonata_api_read"}}
src/Controller/Api/GalleryController.php:163:     *  output={"class"="Sonata\MediaBundle\Model\Media", "groups"={"sonata_api_read"}},
src/Controller/Api/GalleryController.php:195:     *  output={"class"="Sonata\MediaBundle\Model\GalleryHasMedia", "groups"={"sonata_api_read"}},
src/Controller/Api/MediaController.php:76:     *  output={"class"="Sonata\DatagridBundle\Pager\PagerInterface", "groups"={"sonata_api_read"}}
src/Controller/Api/MediaController.php:136:     *  output={"class"="Sonata\MediaBundle\Model\Media", "groups"={"sonata_api_read"}},
src/Controller/Api/MediaController.php:261:     *  output={"class"="Sonata\MediaBundle\Model\Media", "groups"={"sonata_api_read"}},
src/Controller/Api/MediaController.php:299:     *  output={"class"="Sonata\MediaBundle\Model\Media", "groups"={"sonata_api_read"}},
src/Controller/Api/MediaController.php:336:     *  input={"class"="Sonata\MediaBundle\Model\Media", "groups"={"sonata_api_write"}},
src/Controller/Api/MediaController.php:337:     *  output={"class"="Sonata\MediaBundle\Model\Media", "groups"={"sonata_api_read"}},
src/Controller/GalleryAdminController.php:78:        if (!method_exists('Symfony\Bridge\Twig\AppVariable', 'getToken')) {
src/Controller/GalleryAdminController.php:79:            $twig->getExtension('Symfony\Bridge\Twig\Extension\FormExtension')
src/Controller/GalleryAdminController.php:84:        $twig->getRuntime('Symfony\Bridge\Twig\Form\TwigRenderer')->setTheme($formView, $theme);
src/Controller/MediaAdminController.php:122:        if (!method_exists('Symfony\Bridge\Twig\AppVariable', 'getToken')) {
src/Controller/MediaAdminController.php:123:            $twig->getExtension('Symfony\Bridge\Twig\Extension\FormExtension')
src/Controller/MediaAdminController.php:128:        $twig->getRuntime('Symfony\Bridge\Twig\Form\TwigRenderer')->setTheme($formView, $theme);
src/Controller/MediaController.php:148:     * Inject the Symfony\Component\HttpFoundation\Request into the actions instead.
src/DependencyInjection/Configuration.php:430:                        ->scalarNode('media')->defaultValue('Application\\Sonata\\MediaBundle\\Entity\\Media')->end()
src/DependencyInjection/Configuration.php:431:                        ->scalarNode('gallery')->defaultValue('Application\\Sonata\\MediaBundle\\Entity\\Gallery')->end()
src/DependencyInjection/Configuration.php:432:                        ->scalarNode('gallery_has_media')->defaultValue('Application\\Sonata\\MediaBundle\\Entity\\GalleryHasMedia')->end()
src/DependencyInjection/Configuration.php:433:                        ->scalarNode('category')->defaultValue('Application\\Sonata\\ClassificationBundle\\Entity\\Category')->end()
src/DependencyInjection/SonataMediaExtension.php:55:            $amazonS3Definition->setFactory(['Aws\S3\S3Client', 'factory']);
src/DependencyInjection/SonataMediaExtension.php:57:            $amazonS3Definition->setFactoryClass('Aws\S3\S3Client');
src/DependencyInjection/SonataMediaExtension.php:479:                $container->setParameter('sonata.media.adapter.filesystem.opencloud.class', 'OpenCloud\OpenStack');
src/DependencyInjection/SonataMediaExtension.php:482:                $container->setParameter('sonata.media.adapter.filesystem.opencloud.class', 'OpenCloud\Rackspace');
src/DependencyInjection/SonataMediaExtension.php:528:            'Sonata\\MediaBundle\\CDN\\CDNInterface',
src/DependencyInjection/SonataMediaExtension.php:529:            'Sonata\\MediaBundle\\CDN\\CloudFront',
src/DependencyInjection/SonataMediaExtension.php:530:            'Sonata\\MediaBundle\\CDN\\Fallback',
src/DependencyInjection/SonataMediaExtension.php:531:            'Sonata\\MediaBundle\\CDN\\PantherPortal',
src/DependencyInjection/SonataMediaExtension.php:532:            'Sonata\\MediaBundle\\CDN\\Server',
src/DependencyInjection/SonataMediaExtension.php:533:            'Sonata\\MediaBundle\\Extra\\Pixlr',
src/DependencyInjection/SonataMediaExtension.php:534:            'Sonata\\MediaBundle\\Filesystem\\Local',
src/DependencyInjection/SonataMediaExtension.php:535:            'Sonata\\MediaBundle\\Filesystem\\Replicate',
src/DependencyInjection/SonataMediaExtension.php:536:            'Sonata\\MediaBundle\\Generator\\DefaultGenerator',
src/DependencyInjection/SonataMediaExtension.php:537:            'Sonata\\MediaBundle\\Generator\\GeneratorInterface',
src/DependencyInjection/SonataMediaExtension.php:538:            'Sonata\\MediaBundle\\Generator\\ODMGenerator',
src/DependencyInjection/SonataMediaExtension.php:539:            'Sonata\\MediaBundle\\Generator\\PHPCRGenerator',
src/DependencyInjection/SonataMediaExtension.php:540:            'Sonata\\MediaBundle\\Metadata\\AmazonMetadataBuilder',
src/DependencyInjection/SonataMediaExtension.php:541:            'Sonata\\MediaBundle\\Metadata\\MetadataBuilderInterface',
src/DependencyInjection/SonataMediaExtension.php:542:            'Sonata\\MediaBundle\\Metadata\\NoopMetadataBuilder',
src/DependencyInjection/SonataMediaExtension.php:543:            'Sonata\\MediaBundle\\Metadata\\ProxyMetadataBuilder',
src/DependencyInjection/SonataMediaExtension.php:544:            'Sonata\\MediaBundle\\Model\\Gallery',
src/DependencyInjection/SonataMediaExtension.php:545:            'Sonata\\MediaBundle\\Model\\GalleryHasMedia',
src/DependencyInjection/SonataMediaExtension.php:546:            'Sonata\\MediaBundle\\Model\\GalleryHasMediaInterface',
src/DependencyInjection/SonataMediaExtension.php:547:            'Sonata\\MediaBundle\\Model\\GalleryInterface',
src/DependencyInjection/SonataMediaExtension.php:548:            'Sonata\\MediaBundle\\Model\\GalleryManager',
src/DependencyInjection/SonataMediaExtension.php:549:            'Sonata\\MediaBundle\\Model\\GalleryManagerInterface',
src/DependencyInjection/SonataMediaExtension.php:550:            'Sonata\\MediaBundle\\Model\\Media',
src/DependencyInjection/SonataMediaExtension.php:551:            'Sonata\\MediaBundle\\Model\\MediaInterface',
src/DependencyInjection/SonataMediaExtension.php:552:            'Sonata\\MediaBundle\\Model\\MediaManagerInterface',
src/DependencyInjection/SonataMediaExtension.php:553:            'Sonata\\MediaBundle\\Provider\\BaseProvider',
src/DependencyInjection/SonataMediaExtension.php:554:            'Sonata\\MediaBundle\\Provider\\BaseVideoProvider',
src/DependencyInjection/SonataMediaExtension.php:555:            'Sonata\\MediaBundle\\Provider\\DailyMotionProvider',
src/DependencyInjection/SonataMediaExtension.php:556:            'Sonata\\MediaBundle\\Provider\\FileProvider',
src/DependencyInjection/SonataMediaExtension.php:557:            'Sonata\\MediaBundle\\Provider\\ImageProvider',
src/DependencyInjection/SonataMediaExtension.php:558:            'Sonata\\MediaBundle\\Provider\\MediaProviderInterface',
src/DependencyInjection/SonataMediaExtension.php:559:            'Sonata\\MediaBundle\\Provider\\Pool',
src/DependencyInjection/SonataMediaExtension.php:560:            'Sonata\\MediaBundle\\Provider\\VimeoProvider',
src/DependencyInjection/SonataMediaExtension.php:561:            'Sonata\\MediaBundle\\Provider\\YouTubeProvider',
src/DependencyInjection/SonataMediaExtension.php:562:            'Sonata\\MediaBundle\\Resizer\\ResizerInterface',
src/DependencyInjection/SonataMediaExtension.php:563:            'Sonata\\MediaBundle\\Resizer\\SimpleResizer',
src/DependencyInjection/SonataMediaExtension.php:564:            'Sonata\\MediaBundle\\Resizer\\SquareResizer',
src/DependencyInjection/SonataMediaExtension.php:565:            'Sonata\\MediaBundle\\Security\\DownloadStrategyInterface',
src/DependencyInjection/SonataMediaExtension.php:566:            'Sonata\\MediaBundle\\Security\\ForbiddenDownloadStrategy',
src/DependencyInjection/SonataMediaExtension.php:567:            'Sonata\\MediaBundle\\Security\\PublicDownloadStrategy',
src/DependencyInjection/SonataMediaExtension.php:568:            'Sonata\\MediaBundle\\Security\\RolesDownloadStrategy',
src/DependencyInjection/SonataMediaExtension.php:569:            'Sonata\\MediaBundle\\Security\\SessionDownloadStrategy',
src/DependencyInjection/SonataMediaExtension.php:570:            'Sonata\\MediaBundle\\Templating\\Helper\\MediaHelper',
src/DependencyInjection/SonataMediaExtension.php:571:            'Sonata\\MediaBundle\\Thumbnail\\ConsumerThumbnail',
src/DependencyInjection/SonataMediaExtension.php:572:            'Sonata\\MediaBundle\\Thumbnail\\FormatThumbnail',
src/DependencyInjection/SonataMediaExtension.php:573:            'Sonata\\MediaBundle\\Thumbnail\\ThumbnailInterface',
src/DependencyInjection/SonataMediaExtension.php:574:            'Sonata\\MediaBundle\\Twig\\Extension\\MediaExtension',
src/DependencyInjection/SonataMediaExtension.php:602:        return interface_exists('Sonata\ClassificationBundle\Model\CategoryInterface')
src/Document/BaseGallery.php:18: * Bundle\MediaBundle\Document\BaseGallery.
src/Entity/BaseGallery.php:18: * Bundle\MediaBundle\Entity\BaseGallery.
src/Filesystem/Replicate.php:37:     * @param \Gaufrette\Adapter $master
src/Filesystem/Replicate.php:38:     * @param \Gaufrette\Adapter $slave
src/Form/DataTransformer/ProviderDataTransformer.php:124:            // Error message inspired from Monolog\ErrorHandler
src/Form/DataTransformer/ServiceProviderDataTransformer.php:77:            // Error message taken from Monolog\ErrorHandler
src/Form/Type/ApiMediaType.php:111:        // NEXT_MAJOR: Return 'Sonata\MediaBundle\Form\Type\ApiDoctrineMediaType'
src/Form/Type/ApiMediaType.php:113:        return method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')
src/Form/Type/ApiMediaType.php:114:            ? 'Sonata\MediaBundle\Form\Type\ApiDoctrineMediaType'
src/Form/Type/MediaType.php:91:        // NEXT_MAJOR: Remove ternary and keep 'Symfony\Component\Form\Extension\Core\Type\CheckboxType' value.
src/Form/Type/MediaType.php:95:            method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')
src/Form/Type/MediaType.php:96:                ? 'Symfony\Component\Form\Extension\Core\Type\CheckboxType'
src/Form/Type/MediaType.php:146:        if (method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')) {
src/Form/Type/MediaType.php:171:        // NEXT_MAJOR: Return 'Symfony\Component\Form\Extension\Core\Type\FormType'
src/Form/Type/MediaType.php:173:        return method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')
src/Form/Type/MediaType.php:174:            ? 'Symfony\Component\Form\Extension\Core\Type\FormType'
src/Listener/BaseMediaEventSubscriber.php:127:     * @return \Sonata\MediaBundle\Model\MediaInterface
src/Listener/PHPCR/MediaEventSubscriber.php:40:        /* @var $args \Doctrine\Common\Persistence\Event\LifecycleEventArgs */
src/Listener/PHPCR/MediaEventSubscriber.php:41:        /** @var $dm \Doctrine\ODM\PHPCR\DocumentManager */
src/Metadata/ProxyMetadataBuilder.php:82:        if ((!in_array('Gaufrette\Adapter\AmazonS3', $adapterClassNames) && !in_array('Gaufrette\Adapter\AwsS3', $adapterClassNames)) || !$this->container->has('sonata.media.metadata.amazon')) {
src/Model/MediaManagerInterface.php:20: * @deprecated Use Sonata\CoreBundle\Model\ManagerInterface instead
src/Model/Media.php:162:            if (null !== $value && !is_a($value, 'Sonata\ClassificationBundle\Model\CategoryInterface')) {
src/Model/Media.php:164:                    '$category should be an instance of Sonata\ClassificationBundle\Model\CategoryInterface or null'
src/Model/Media.php:647:        if (class_exists('Symfony\Component\Validator\Constraints\Expression')) {
src/Model/Media.php:652:        $metadata->addConstraint(new Assert\Callback($method));
src/Model/Media.php:663:                throw new \InvalidArgumentException('Argument 1 should be an instance of Symfony\Component\Validator\ExecutionContextInterface or Symfony\Component\Validator\Context\ExecutionContextInterface');
src/PHPCR/BaseGallery.php:18: * Bundle\MediaBundle\Document\BaseGallery.
src/PHPCR/BaseGallery.php:32:        $this->galleryHasMedias = new \Doctrine\Common\Collections\ArrayCollection();
src/Provider/BaseVideoProvider.php:134:            // NEXT_MAJOR: Remove ternary and keep 'Symfony\Component\Form\Extension\Core\Type\TextType' value
src/Provider/BaseVideoProvider.php:136:            method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')
src/Provider/BaseVideoProvider.php:137:                ? 'Symfony\Component\Form\Extension\Core\Type\TextType'
src/Provider/BaseVideoProvider.php:149:            // NEXT_MAJOR: Remove ternary and keep 'Symfony\Component\Form\Extension\Core\Type\TextType' value
src/Provider/BaseVideoProvider.php:151:            method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')
src/Provider/BaseVideoProvider.php:152:                ? 'Symfony\Component\Form\Extension\Core\Type\TextType'
src/Provider/BaseVideoProvider.php:168:        // NEXT_MAJOR: Remove ternary and keep 'Symfony\Component\Form\Extension\Core\Type\TextType'
src/Provider/BaseVideoProvider.php:172:            method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')
src/Provider/BaseVideoProvider.php:173:                ? 'Symfony\Component\Form\Extension\Core\Type\TextType'
src/Provider/DailyMotionProvider.php:152:        if (preg_match("/www.dailymotion.com\/video\/([0-9a-zA-Z]*)_/", $media->getBinaryContent(), $matches)) {
src/Provider/FileProvider.php:102:            // NEXT_MAJOR: Remove ternary and keep 'Symfony\Component\Form\Extension\Core\Type\FileType' value
src/Provider/FileProvider.php:104:            method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')
src/Provider/FileProvider.php:105:                ? 'Symfony\Component\Form\Extension\Core\Type\FileType'
src/Provider/FileProvider.php:118:            // NEXT_MAJOR: Remove ternary and keep 'Symfony\Component\Form\Extension\Core\Type\FileType' value
src/Provider/FileProvider.php:120:            method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')
src/Provider/FileProvider.php:121:                ? 'Symfony\Component\Form\Extension\Core\Type\FileType'
src/Provider/FileProvider.php:137:        // NEXT_MAJOR: Remove $fileType variable and inline 'Symfony\Component\Form\Extension\Core\Type\FileType'
src/Provider/FileProvider.php:139:        $fileType = method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')
src/Provider/FileProvider.php:140:            ? 'Symfony\Component\Form\Extension\Core\Type\FileType'
src/Provider/FileProvider.php:285:        if (!$this->getFilesystem()->getAdapter() instanceof \Sonata\MediaBundle\Filesystem\Local) {
src/Provider/FileProvider.php:378:            throw new \RuntimeException('Please define a valid media\'s name');
src/Provider/MediaProviderInterface.php:71:     * @return \Gaufrette\File
src/Provider/VimeoProvider.php:145:        if (preg_match("/vimeo\.com\/(video\/|)(\d+)/", $media->getBinaryContent(), $matches)) {
src/Provider/YouTubeProvider.php:261:        if (preg_match("/^(?:http(?:s)?:\/\/)?(?:www\.)?(?:m\.)?(?:youtu\.be\/|youtube\.com\/(?:(?:watch)?\?(?:.*&)?v(?:i)?=|(?:embed|v|vi|user)\/))([^\#\?&\"'>]+)/", $media->getBinaryContent(), $matches)) {
src/Security/RolesDownloadStrategy.php:46:            throw new \InvalidArgumentException('Argument 2 should be an instance of Symfony\Component\Security\Core\Authorization\AuthorizationCheckerInterface or Symfony\Component\Security\Core\SecurityContextInterface');
src/Security/SessionDownloadStrategy.php:63:                'Using an instance of Symfony\Component\DependencyInjection\ContainerInterface is deprecated since 
src/Security/SessionDownloadStrategy.php:65:                Use Symfony\Component\HttpFoundation\Session\SessionInterface instead.',
src/Security/SessionDownloadStrategy.php:74:                '$session should be an instance of Symfony\Component\HttpFoundation\Session\SessionInterface'
src/SonataMediaBundle.php:54:            'sonata_media_type' => 'Sonata\MediaBundle\Form\Type\MediaType',
src/SonataMediaBundle.php:55:            'sonata_media_api_form_media' => 'Sonata\MediaBundle\Form\Type\ApiMediaType',
src/SonataMediaBundle.php:56:            'sonata_media_api_form_doctrine_media' => 'Sonata\MediaBundle\Form\Type\ApiDoctrineMediaType',
src/SonataMediaBundle.php:57:            'sonata_media_api_form_gallery' => 'Sonata\MediaBundle\Form\Type\ApiGalleryType',
src/SonataMediaBundle.php:58:            'sonata_media_api_form_gallery_has_media' => 'Sonata\MediaBundle\Form\Type\ApiGalleryHasMediaType',
src/Twig/Extension/FormatterMediaExtension.php:52:            'Sonata\MediaBundle\Model\MediaInterface' => [

```
</details>

**Comment by greg0ire** [comment]
This could help: https://github.com/Symplify/Symplify/pull/421

**Comment by greg0ire** [comment]
Hmm no in fact, it's not related to `::class`

**Comment by jordisala1991** [comment]
Already fixed in #1355 . Closing
